tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,122 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Plex
+finish-args:
+  - --socket=x11
+  # currently now working - --socket=wayland
```

Comment:
do you mean currently not working here?

Reply:
Currently not working because we missed shipping the Qt wayland plugin. Will be corrected in the next release, but until then, the application doesn't run under wayland.

Reply:
that's fine - it's just that the comment says 'now working' which means the opposite. Fine to add it next release, when you do switch x11 to fallback-x11 to get wayland picked by default if on a wayland native system (it will have no effect on an x11 native system which will continue to use x11 correctly).

Reply:
Oh! I misread your question and didn't notice the typo until now. I'll fix that comment to avoid confusion.